.github/workflows/release-common.yaml

```
@@ -27,33 +27,28 @@ jobs:
 
-      - name: Extract Python version from pyproject.toml
+      - name: Extract Python version from .python-version
         id: extract-python
         run: |
-          PYTHON_SPEC=$(grep -E '^[[:space:]]*python[[:space:]]*=' pyproject.toml | sed -E 's/.*=["'\''](.*)["'\'']/\1/')
-          PYTHON_VERSION=$(echo "$PYTHON_SPEC" | grep -oE '[0-9]+\.[0-9]+' | head -n1)
+          PYTHON_VERSION=$(head -n 1 .python-version)
           echo "version=$PYTHON_VERSION" >> "$GITHUB_OUTPUT"
 
       - name: Set up Python
         uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)